### DISCUSSION TITLE
今天又不能自动打卡了？

### DISCUSSION THREAD

**Discussion by WarrenLee0726** [lxysl/Auto_Attendance#34] [General]
前几天更新了代码之后，可以正常打卡2天，今天貌似又不能正常打卡了。
请问大家有好的解决办法吗？

**Comment by M-2Pac** [comment]
https://github.com/hidragonma/Auto_Attendance
这个老哥更新代码了，可以打卡，应该是动态验证码的问题

**Comment by zhwdzh** [comment]
前面有人说了，你把captcha后面的value属性改为name就可以正常运行了

**Comment by lxysl** [comment]
已参考 https://github.com/hidragonma/Auto_Attendance 修复
